### ISSUE TITLE
charts/core: values.yml for testnet and pubnet

### ISSUE THREAD

**Issue by sreuland** [stellar/helm-charts#65]
### What problem does your feature solve?
if you want to install the helm chart for core on testnet or pubnet, there is no reference values.yml to use, which aggregates many specific settings needed for network like validators, passphrase, archive urls, etc. it would take a long time and be error prone to try and set that up manually.

### What would you like to see?
`testnet_values.yml` and `pubnet_values.yml` are created in the core chart path to be usable in `helm install --values`.
Or `global.network=[testnet|pubnet]`  works the same and prevents need for `testnet_values.yml` and `pubnet_values.yml`  .

updated docs can refer to correct config - https://github.com/stellar/stellar-docs/pull/216

### What alternatives are there?
users take the futurenet_values.yml and manually figure out the changes from there for testnet or pubnet.

**Comment by sreuland** [comment]
is this request worthwhile for consideration? if so, is there an estimate on when it will schedule, there is a docs draft that would reference the availability of the new values.yml if they exist.

**Comment by sreuland** [comment]
these values files have been added - https://github.com/stellar/helm-charts/pull/78

**Comment by sreuland** [comment]
I mistakenly closed, reopening, this is for network values yml's for core chart, not the rpc chart, which was done on https://github.com/stellar/helm-charts/pull/78

**Comment by sreuland** [comment]
> is this request worthwhile for consideration? if so, is there an estimate on when it will schedule, there is a docs draft pr, [stellar/stellar-docs#216](https://github.com/stellar/stellar-docs/pull/216), that would reference the availability of the new values.yml if they exist.

@jacekn , do you think the values yml files created for captive core config on rpc https://github.com/stellar/helm-charts/pull/78, could be leveraged, copied in to this effort?

**Comment by jacekn** [comment]
@sreuland yeah copying the values might be a good idea. I believe both charts should be using the same config format.
I'll think about deduplication too - maybe there will be a way to maintain single copy of pubnet and testnet quorumsets.
